src/wasm/wasm-binary.cpp

```diff
@@ -3160,86 +3160,72 @@ void WasmBinaryBuilder::readMemoryAccess(Address& alignment, Address& offset) {
 bool WasmBinaryBuilder::maybeVisitLoad(Expression*& out,
                                        uint8_t code,
                                        bool isAtomic) {
-  Load* curr;
+  auto* curr = allocator.alloc<Load>();
```

Comment:
Isn't this going to frequently leak an unused `Load`?

Reply:
Oh, heh, how silly... yes, I misread the code around this. Fix in a sec.